frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -690,7 +690,7 @@ export default function InteractiveWaveformModal({
     )[0];
     setPulse3dVersionEOLDate(
       selectedVersionMetadata.end_of_life_date
-        ? ` Version ${selectedVersionMetadata.version} will be removed after ${electedVersionMetaData.end_of_life_date}.`
+        ? ` Version ${selectedVersionMetadata.version} will be removed after ${selectedVersionMetaData.end_of_life_date}.`
```

Comment:
Shouldn't the `d` in `metadata` also be lowercase?

Reply:
yes